Changes

```diff
@@ -2,6 +2,11 @@ Revision history for WebService-PayPal-PaymentsAdvanced
 
 {{$NEXT}}
 
+0.000022 2017-05-08
```

Comment:
This line is unnecessary. It is automatically generated by dzil on release. We should probably do a release when this PR is merged.